### PR TITLE
Updates some packages to use git-checkout (batch 16)

### PR DESCRIPTION
**Pull request by dakaneye**
Packages:
* net-tools
* nghttp2
* nginx-mainline
* nginx-stable
* npth

### CODE REVIEW

File: netcat-openbsd.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package:
   name: netcat-openbsd
-  version: "1.226"
```
Comment (dakaneye) [inline netcat-openbsd.yaml:3]: Are there any other consequences from changing this versioning scheme? release-monitoring.org doesn't have the -1.1 suffix but the repo does, wasn't sure which to use.

File: nginx-mainline.yaml [hunk 2]
```diff
@@ -108,7 +111,8 @@ pipeline:
           --with-stream_realip_module \
           --with-stream_geoip_module=dynamic \
           --with-stream_ssl_preread_module
-  - runs: |
+  - working-directory: nginx-mainline
```
Comment (smoser) [inline nginx-mainline.yaml:114]: What does the word 'mainline' mean to you?
I would have interpreted this as upstream/main .

maybe `nginx-git` or `nginx-checkout` or `git-checkout` ?

**Review by smoser — CHANGES_REQUESTED** [review]
for nginx, its not useful to the reader to say 'mainline' or 'stable'. I'd suggest making them the same 'nginx-checkout' to reduce differences between the two files.

Feel free to disagree and i can probably be swayed. 

thank you.

File: nginx-mainline.yaml (hunk 2 above)
Comment (dakaneye) [reply nginx-mainline.yaml:114]: AFAICT it refers to the version stream of the package. I was just trying to be consistent with the package naming scheme.

Comment (smoser) [reply nginx-mainline.yaml:114]: i hadn't even noticed this was `nginx-mainline.yaml`.

**Review by smoser — APPROVED** [review]

**Pull request merged**